### ISSUE TITLE
artist biographies 

### ISSUE THREAD

**Issue by zettaexa** [Malopieds/InnerTune#188]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

a section of the artist page that details the musician through integration to wikipedia or RYM's artist biographies 

### Why do you want this feature?

so you can read about the artist's genres and facts while listening, just a good past time to do

### Additional information

by rym biographies i mean these rough guides  idk if you can achieve them through its API
![Screenshot_20240710-033414](https://github.com/Malopieds/InnerTune/assets/79076739/89581c51-d9b1-4aa3-b574-f07e8a26f1bc)

**Comment by Malopieds** [comment]
Yes YouTube music have for some artists in the API, I would look into it

**Comment by zettaexa** [comment]
> Yes YouTube music have for some artists in the API, I would look into it

You can also fetch Wikipedia for artist biographies !! Im sure it's free if you give credit at the end
